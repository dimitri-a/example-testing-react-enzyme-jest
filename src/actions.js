export const handleStartCar = () => {
  console.log('car has started');
};
