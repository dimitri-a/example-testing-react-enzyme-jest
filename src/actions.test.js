import * as actions from './actions';

describe('actions', () => {
  describe('handleStartCar', () => {
    it('returns type HANDLE_START_CAR with payload', function () {
      const action = actions.handleStartCar();
      expect(action).toMatchSnapshot();
    });
  });
});
