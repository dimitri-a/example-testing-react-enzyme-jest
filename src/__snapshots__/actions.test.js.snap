// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions handleStartCar returns type HANDLE_START_CAR with payload 1`] = `
Object {
  "type": "HANDLE_START_CAR",
}
`;
